image-layout.md

```diff
@@ -162,7 +162,7 @@ Those tags will often be represented in an image-layout repository with matching
 
 ### Index Example
 
-```json,title=Image%20Index&mediatype=application/vnd.oci.image.index.v1%2Bjson
+```json
```

Comment:
I remove this title as not to valid it. I understand that you want to valid it. 
But I think `spec_test.go` should only valid those who are announced only in itself *.md. So I remove it, for it is just a reference. It is IMO, to be not sure.